app/routines/newflow/ensure_unverified_user.rb

```diff
@@ -0,0 +1,25 @@
+# Users who started signing up in the old flow but never filled out their profile information
+# are in the state 'needs_profile'. In the new flow, that translates to 'unverified'.
+# So we update their state to match the new flow.
+module Newflow
+  class EnsureUnverifiedUser
+    lev_routine
+
+    def exec(user)
+      if user.state == 'needs_profile'
```

Comment:
I would def recommend putting strings like this in one place, as constants.  scattering them around like this is error prone and makes the code difficult to change.